pinot-spi/src/main/java/org/apache/pinot/spi/config/table/SegmentsValidationAndRetentionConfig.java

```diff
@@ -29,6 +29,8 @@
 public class SegmentsValidationAndRetentionConfig extends BaseJsonConfig {
   private String _retentionTimeUnit;
   private String _retentionTimeValue;
+  private String _deletedSegmentRetentionTimeUnit;
```

Comment:
Suggest changing it to `_deletedSegmentRetenionPeriod` for easier configuration. Period can be parsed using `TimeUtils.convertPeriodToMillis()`

Reply:
would like to keep this consistent with the `retentionTime` setting for now, wdyt